### PR TITLE
Update color variables on apps/src/accounts stylesheets

### PR DESCRIPTION
**Pull request by kelbyhawn**
These two stylesheets need to be updated to use `primitiveColors.css` explicitly to not rely on the bundling of other imported stylesheets. This is causing drone failures on this PR: https://github.com/code-dot-org/code-dot-org/pull/62863. 

## Links
Jira ticket: [DESIGN2-206](https://codedotorg.atlassian.net/browse/DESIGN2-206)
Slack convo: [here](https://codedotorg.slack.com/archives/C07UW4ED66Q/p1733767517339399?thread_ts=1733521215.903719&cid=C07UW4ED66Q)

## Testing story
Tested locally

### CODE REVIEW

File: apps/src/accounts/common/common.styles.module.scss [hunk 1]
```diff
@@ -31,6 +32,6 @@ button.submit {
 }
 
 h2.sectionHeader {
-  color: $light_primary_500;
+  color: var(--brand-teal-50);
```
Comment (kelbyhawn) [inline apps/src/accounts/common/common.styles.module.scss:35]: Screenshot 2024-12-09 at 10 34 20 AM

File: apps/src/accounts/AccountInformation/style.module.scss [hunk 2]
```diff
@@ -9,7 +10,7 @@
 
     &.verifiedTeacher {
       font-weight: bold;
-      color: $light_affirmative_500;
+      color: var(--sentiment-success-50);
```
Comment (kelbyhawn) [inline apps/src/accounts/AccountInformation/style.module.scss:13]: Screenshot 2024-12-09 at 10 53 08 AM

**Review by drizco — APPROVED** [review]
lgtm!

**Review by levadadenys — COMMENTED** [review]
Thank you!

**Pull request merged**